### ISSUE TITLE
Tekken 7 Shutdown after splashscreen (389730)

### ISSUE THREAD

**Issue by victuxbb** [ValveSoftware/Proton#2100]
# Compatibility Report
- Name of the game with compatibility issues: Tekken 7
- Steam AppID of the game: 389730

## System Information
- GPU: GTX 1080 TI
- Driver/LLVM version: nvidia 410.78
- Kernel version:  4.15.0-42
- Link to full system information report as [Gist](https://gist.github.com/victuxbb/4fa84659eb6c3d74b729f655435da52f):
- Proton version: 3.16-5 Beta

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-389730.log](https://github.com/ValveSoftware/Proton/files/2683574/steam-389730.log)

## Symptoms
After the initial splashscreen with a logo in the bottom right corner the game suddenly shutdown returning to steam with no message error.

## Reproduction
Just launch the game from steam

Seems related with this issue, but I already tried to reinstall the game twice and always happens the same.
https://github.com/ValveSoftware/Proton/issues/278
Also this issue https://github.com/ValveSoftware/Proton/issues/1982 seems related, I'm using Ubuntu 18.04 that uses gnome-shell.

**Comment by ghost** [comment]
I take it pre-16... 7.x doesn't have this problem?

The log here and in the other issues show far too many problems for my liking such as:
`7970.389:0028:002f:fixme:wer:WerReportSetParameter (0x208f4ac0, 6, L"Exception Code", L"c0000005") :stub`
`7970.386:0028:002f:trace:seh:NtRaiseException code=c0000005 flags=0 addr=0x7bc62fb8 ip=7bc62fb8 tid=002f`

There's 630 "Exception" in your log alone though difficult to say if they are related to the game not starting since all the known logs are of it broken. But, c0000005 even on Windows is not something you want to see.

**Comment by ATypescriptEnjoyer** [comment]
Just launching the game most likely won't be reproducable for others, I've been able to run Tekken 7 on both versions of Proton before, so there may be an issue with your setup instead. If you want I can dump my logs when I get home if it may help you debug this issue.

**Comment by victuxbb** [comment]
@JackRyder you are right, but I really don't know what to say, I just have a regular Ubuntu 18.04
I thought about my "special" resolution of 3440x1440 but I changed it and it continues failing.
Doing a grep in my log file there is a lot of lines like this with different dll's:
```cannot open .so lib for builtin L"steam_api64.dll": /media/gaming/Steam/steamapps/common/Proton 3.16 Beta/dist/lib/wine/steam_api64.dll.so: cannot open shared object file: No such file or directory```

Another special thing in my setup is that I have all games installed in another harddrive under /media/gaming.

Anyway, thanks for the help!

**Comment by victuxbb** [comment]
@byte1024 So...any idea where this exceptions with "c0000005" come from?

**Comment by ATypescriptEnjoyer** [comment]
`c0000005` is a memory violation, might be a bad memory pointer because of your game crashing? When it crashes is it still visible in your task manager? I am also using Ubuntu 18.04 and I play at 2k resolution. Can you get a copy of your `/etc/fstab` and drop it in a gist? that may help as I know games have been acting strange with certain file system types.

**Comment by victuxbb** [comment]
Yeah, of course:
https://gist.github.com/victuxbb/3389e65ec60e8a6d4b7bbfd060a1d8e6

**Comment by alasky17** [comment]
@victuxbb Could you run Steam from the terminal and post the output?  To do this, quit Steam completely.  Then just "steam" in the terminal should run Steam and spit out some error into the running terminal.

**Comment by ATypescriptEnjoyer** [comment]
fstab looks good, i'll check my logs when I get home, see if there's anything noticable.

**Comment by victuxbb** [comment]
Definitively, should be something strange in the setup of my machine:
* Dark Souls 3
* Spacelords
* Tekken 7

Happens exactly the same, after a first splashscreen the game disappear returning to steam client without error.
The exception is The Witcher 3, works perfect without any issue :(

**Comment by ATypescriptEnjoyer** [comment]
I imagine you have, but did you go through here and install all the requirements without error? https://github.com/ValveSoftware/Proton/wiki/Requirements

Also some reviews on the game from others here, some suggest using D3D11 https://www.protondb.com/app/389730

**Comment by JackRyder** [comment]
Also some reviews on the game from others here, some suggest using D3D11 https://www.protondb.com/app/389730

**Comment by victuxbb** [comment]
Ok, everything working perfect now!
The problem was that before the release of "Steam play" I was using wine to run DOOM and Withcher3.
Seems that in some way Proton is relying on some configuration or something of the wine installed on the system so I guess is not completely isolated of the system.
Finally, I formated my system and I've installed Ubuntu 18.04 again and everything works as expected :)
In conclusion, your old installations of wine, lutris, etc can be a problem.

I hope this information can be useful to someone!
@JackRyder @byte1024 @kisak-valve Thanks for your time!

**Comment by alasky17** [comment]
Just in case anyone else is seeing a similar problem in the future -- if you have wine or a variation of wine installed, check your environment variables to see if you set any environment variables set for wine.  These might be causing the trouble.

**Comment by llGenesis** [comment]
i have the exact Problem but none of the fixes here or protondb helped so far i Tried PROTON_NO_FSYNC=1, taskset -c 0-4, gamemoderun. I also tried and swiched to other versions of Proton but to no avial. I saw a fix for windows that i had to delete my save to get it working again but that also didnt fix the issue.
my System info : https://gist.github.com/llGenesis/6b37abc28e034f33bf72bf78a4acb0d2
proton log: https://gist.github.com/llGenesis/52e5680da145cae297a7a1f5ce058930
